src/sentry/runner/commands/createuser.py

```diff
@@ -68,23 +80,31 @@ def createuser(email, password, superuser, no_password, no_input, force_update):
         if superuser is None:
             superuser = _get_superuser()
 
+        # for self hosted to give superusers admin permissions
+        if superuser and settings.SENTRY_SELF_HOSTED and superadmin is None:
+            superadmin = _get_superadmin()
+
+        if staff is None:
+            staff = _get_staff()
```

Comment:
You should still check for `no_input` and not prompt for these if it is `True`?

Reply:
All these ifs are covered by the check herehttps://github.com/getsentry/sentry/pull/32802/files#diff-4c81db7e7d333c1766ce5cac76b775aae663f2f7ea0744fa44d75c92855b4331R73

Reply:
ah, sorry missed that 😅